xml/System.ServiceModel.Channels/ServiceChannelProxy.xml

```diff
@@ -1,1536 +1,1540 @@
+      <MemberType>Property</MemberType>
+      <Implements>
+        <InterfaceMember>P:System.ServiceModel.IClientChannel.Via</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Uri</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets the URI that contains the transport address to which messages are sent on the client channel.</summary>
+        <value>The <see cref="T:System.Uri" /> that contains the transport address to which messages are sent on the channel.</value>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Channels.ServiceChannelProxy&gt; instance is cast to an &lt;xref:System.ServiceModel.IClientChannel&gt; interface.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.ICommunicationObject.Abort">
+      <MemberSignature Language="C#" Value="void ICommunicationObject.Abort ();" />
+      <MemberSignature Language="ILAsm" Value=".method hidebysig newslot virtual instance void System.ServiceModel.ICommunicationObject.Abort() cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.ServiceModel.Channels.ServiceChannelProxy.System#ServiceModel#ICommunicationObject#Abort" />
+      <MemberSignature Language="VB.NET" Value="Sub Abort () Implements ICommunicationObject.Abort" />
+      <MemberSignature Language="F#" Value="abstract member System.ServiceModel.ICommunicationObject.Abort : unit -&gt; unit&#xA;override this.System.ServiceModel.ICommunicationObject.Abort : unit -&gt; unit" Usage="serviceChannelProxy.System.ServiceModel.ICommunicationObject.Abort " />
+      <MemberSignature Language="C++ CLI" Value=" virtual void System.ServiceModel.ICommunicationObject.Abort() = System::ServiceModel::ICommunicationObject::Abort;" />
+      <MemberType>Method</MemberType>
+      <Implements>
+        <InterfaceMember>M:System.ServiceModel.ICommunicationObject.Abort</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters />
+      <Docs>
+        <summary>Causes a communication object to transition immediately from its current state into the closed state.</summary>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Channels.ServiceChannelProxy&gt; instance is cast to an &lt;xref:System.ServiceModel.ICommunicationObject&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Channels.ServiceChannelProxy"/> instance is cast to an <see cref="T:System.ServiceModel.ICommunicationObject"/> interface.</remarks>

```